site/docs/layouts/_default/single.html

```diff
@@ -13,7 +13,14 @@
         </aside>
         <main id="main" role="main" class="markdown">
           {{ if .Title }}<h1 class="title">{{ .Title }}</h1>{{ end }}
-          {{ .Content }}
+          <!-- The regex finds header tags and converts the content into a link
+            with a link icon appended.
+          -->
+          {{ .Content
+            | replaceRE "(<h[1-6] id=\"(.+)\".*>)(.*)(</h[1-6]>)"
```

Comment:
I think we are safe here as long as there are no strange `>` or `"` characters around.
At least `>` shouldn't appear randomly since that would be invalid HTML, right?
If the current doc renders fine, I'm happy with it :-)

Reply:
Yeah that was my assumption. I think it is particularly safe because the html is generated by a tool (Hugo).